noty-android/data/remote/src/main/java/dev/shreyaspatil/noty/data/remote/api/NotyService.kt

```diff
@@ -17,15 +17,11 @@
 package dev.shreyaspatil.noty.data.remote.api
 
 import dev.shreyaspatil.noty.data.remote.model.request.NoteRequest
+import dev.shreyaspatil.noty.data.remote.model.request.NoteUpdatePinRequest
 import dev.shreyaspatil.noty.data.remote.model.response.NoteResponse
 import dev.shreyaspatil.noty.data.remote.model.response.NotesResponse
 import retrofit2.Response
-import retrofit2.http.Body
-import retrofit2.http.DELETE
-import retrofit2.http.GET
-import retrofit2.http.POST
-import retrofit2.http.PUT
-import retrofit2.http.Path
+import retrofit2.http.*
```

Comment:
@tyaporush can you please revert this change? It's better to have no-wildcard imports

Reply:
@PatilShreyas Removed Wildcard imports